addons/sale/migrations/10.0.1.0/pre-migration.py

```diff
@@ -92,20 +92,20 @@ def warning_update_module_names_partial(cr):
         SELECT id
         FROM ir_model_fields
         WHERE model IN ('res.partner', 'product.template') AND name in %s
-    """, (moved_fields,))
+    """, (tuple(moved_fields),))
     field_ids = [r[0] for r in cr.fetchall()]
```

Comment:
this check happpens in https://github.com/OCA/OpenUpgrade/blob/10.0/.travis.yml#L64 - so either use one of the keywords this greps for deleted addons too or change the regex to output also deleted addons